1.8/usage/service-discovery/load-balancing-vips/overlay-networks.md

```diff
@@ -25,9 +20,13 @@ First, you or the data center operator needs to [configure the overlay networks]
 
 Overlay networks are configured at install time. You or the data center operator will specify a canonical name for each network in the `config.yaml`. When your service needs to launch a container, refer to it by that canonical name.
 
+To use an overlay network in a Marathon app definition, specify the `"network": "USER"` property along with an `ipAddress` field in the form: `{"ipAddress": {"network": "$MYNETWORK"}}`. The value of `$MYNETWORK` is the canonical name of the network.
+
+**Note:** Marathon currently only supports overlay networking through the Docker containerizer (DNI), not the Mesos containerizer.
```

Comment:
Actually, it turns out both are supported now. So goodby to this sentence entirely!